doc/whats_new.rst

```diff
@@ -57,6 +57,8 @@ Changelog
 
     - Add interactive plotting of single trials by right clicking on channel name in epochs browser by `Jaakko Leppakangas`_
 
+    - New logos and logo generation script by `Daniel McCloy`_
```

Comment:
I wouldn't really add this to whats_new because this isn't something the user needs to know